tools/buildHelper/buildHelper.psm1

```diff
@@ -2,35 +2,59 @@
 # Licensed under the MIT License.
 
 # Gets the Current version of PowerShell from the PowerShell repo
+# or formats the version based on the parameters
 function Get-PowerShellVersion
 {
     [CmdletBinding(DefaultParameterSetName='Default')]
     param(
-        [Parameter(ParameterSetName='Preview', HelpMessage="Gets the preview version.  Without this it gets the current stable version.")]
+        [Parameter(Mandatory, ParameterSetName="explicitVersionPreview", HelpMessage="Gets the preview version.  Without this it gets the current stable version.")]
```

Comment:
Style nit: `explicitVersionPreview` -> `ExplicitVersionPreview`

Reply:
Same style nit applies to all the other not-quite-Pascal-case parameter set names